File: train-estimator-main/java/src/test/java/org/katas/HelloTest.java
package org.katas;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class HelloTest {

    @Test
    void shouldWork() {
        assertEquals(3, 1+2);
    }

    @Test
    void print_shouldFail() {
        var hello = new Hello();
        Assertions.assertEquals("Hello, world!", hello.print());
    }
}

File: train-estimator-main/kotlin/src/test/kotlin/org/katas/HelloTest.kt
package org.katas

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class HelloTest {
    @Test
    fun `should work`() {
        assertEquals(3, 1 + 2)
    }

    @Test
    fun `should fail`() {
        assertEquals("Hello world!", Hello().sayHello())
    }
}